django/contrib/postgres/forms/ranges.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, base_widget, attrs=None):
 
     def decompress(self, value):
         if value:
+            if isinstance(value, tuple):
+                  return value
```

Comment:
We should also handle lists.
```suggestion
            if isinstance(value, (list, tuple)):
                  return (value[0], value[1])
```